### PR TITLE
Stop preview auto updates

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
#### Summary of Changes (What does this PR do?)
- [Stops auto update for deploy previews](https://github.com/airqo-platform/AirQo-frontend/pull/941/commits/6d312608e9efc4ecaf646a1620b55f6e2e21448c)
- [Adds missing helm autogenerate files](https://github.com/airqo-platform/AirQo-frontend/pull/941/commits/963a82b93b2a4db9fd52edf10feabf015c673cce)

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

### CODE REVIEW

File: .github/workflows/deploy-frontend-pr-previews.yml [hunk 1]
```diff
@@ -222,6 +210,8 @@ jobs:
           cd calibrate/
           docker build --tag ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/pr-previews/calibrate-app-pr-previews:${{ github.sha }} .
           docker push ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/pr-previews/calibrate-app-pr-previews:${{ github.sha }}
+          docker tag gcr.io/airqo-250220/pr-previews/calibrate-app-pr-previews:${{ github.sha }} gcr.io/airqo-250220/pr-previews/calibrate-app-pr-previews:latest
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontend-pr-previews.yml:213]: Add latest tag to be used in development

File: .github/workflows/deploy-frontend-pr-previews.yml [hunk 2]
```diff
@@ -136,20 +138,6 @@ jobs:
...
-          updateFile: true
-          branch: deployment/argocd
-          message: "Update (dev)netmanager image for PR:${{ needs.branch-name.outputs.lowercase }}"
-          createPR: false
-          changes: |
-            {
-              "image.repository": "${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/pr-previews/netmanager-pr-previews",
-              "image.tag": "${{ github.sha }}"
-            }
-
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontend-pr-previews.yml:152]: Remove auto image tag update for deploy previews

File: k8s/calibrate/values-dev.yaml [hunk 3]
```diff
@@ -5,7 +5,7 @@
 replicaCount: 1
 
 image:
-  repository: gcr.io/airqo-250220/airqo-dev-calibrate-app
+  repository: gcr.io/airqo-250220/pr-previews/calibrate-app-pr-previews
```
Comment (123MwanjeMike) [inline k8s/calibrate/values-dev.yaml:8]: Actual pr-preview image name

File: .github/workflows/deploy-frontend-pr-previews.yml [hunk 4]
```diff
@@ -315,6 +291,8 @@ jobs:
           cd platform/
           docker build --tag ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/pr-previews/next-platform-pr-previews:${{ github.sha }} .
           docker push ${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/pr-previews/next-platform-pr-previews:${{ github.sha }}
+          docker tag gcr.io/airqo-250220/pr-previews/next-platform-pr-previews:${{ github.sha }} gcr.io/airqo-250220/pr-previews/next-platform-pr-previews:latest
```
Comment (Baalmart) [inline .github/workflows/deploy-frontend-pr-previews.yml:294]: Why are we letting of of using secrets?

File: k8s/docs/values-stage.yaml [hunk 5]
```diff
@@ -1,13 +1,9 @@
...
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
 replicaCount: 1
 
 image:
   repository: gcr.io/airqo-250220/airqo-stage-docs
   pullPolicy: Always
-  tag: latest
+  tag: stage-090ad16f-1667922055
```
Comment (Baalmart) [inline k8s/docs/values-stage.yaml:6]: What is the motivation for these tags?

File: k8s/calibrate/values-dev.yaml (hunk 3 above)
Comment (Baalmart) [reply k8s/calibrate/values-dev.yaml:8]: Thanks, looks good to me. Thanks!

**Review by Baalmart — COMMENTED** [review]
Thanks @123MwanjeMike , this is great work!

I guess some review comments in another PR still stand for this PR.
I hope to appreciate sooner rather than later.

File: .github/workflows/deploy-frontend-pr-previews.yml (hunk 4 above)
Comment (123MwanjeMike) [reply .github/workflows/deploy-frontend-pr-previews.yml:294]: Hi @Baalmart 
The image url is not really a secret, or worth being added as a secret.
1. These urls are already in all the k8s manifest files.
2. They are private
3. None owners still can not access them without the required credentials.

File: k8s/docs/values-stage.yaml (hunk 5 above)
Comment (123MwanjeMike) [reply k8s/docs/values-stage.yaml:6]: @Baalmart, we have an auto update bot that updates those images.
This is important to trigger ArgoCD as it only watches for changes in the helm files.

Comment (Baalmart) [reply k8s/docs/values-stage.yaml:6]: Alright @123MwanjeMike , thanks for the clarifications.

Is this related to image versioning? If yes, what is unique about this tag number and how is differentiated from "previous" tags?

File: .github/workflows/deploy-frontend-pr-previews.yml [hunk 6]
```diff
@@ -5,6 +5,10 @@ on:
     branches:
       - staging
 
+env:
+  REGISTRY_URL: gcr.io
+  PROJECT_ID: airqo-250220
+
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-frontend-pr-previews.yml:11]: Adding image registry uri global variables

**Review by 123MwanjeMike — COMMENTED** [review]
Hi @Baalmart & @NoahNsimbe 
I've updated the PR changes based on the feedback provided on the recent chat on this. Kindly review again.

File: .github/workflows/deploy-frontend-pr-previews.yml (hunk 4 above)
Comment (Baalmart) [reply .github/workflows/deploy-frontend-pr-previews.yml]: Thanks @123MwanjeMike for the explanations.

My guess is that we are still following through with the suggestions from the recent discussion about using "global variables"?

File: .github/workflows/deploy-frontend-pr-previews.yml (hunk 6 above)
Comment (123MwanjeMike) [reply .github/workflows/deploy-frontend-pr-previews.yml:11]: @Baalmart, here are the latest changes

**Review by NoahNsimbe — APPROVED** [review]
Thanks @123MwanjeMike
LGTM

File: .github/workflows/deploy-frontend-pr-previews.yml (hunk 6 above)
Comment (Baalmart) [reply .github/workflows/deploy-frontend-pr-previews.yml:11]: Awesome, thanks @123MwanjeMike !

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @123MwanjeMike

File: k8s/netmanager/values-dev.yaml [hunk 7]
```diff
@@ -5,7 +5,7 @@
 replicaCount: 1
 
 image:
-  repository: airqo-dev-platform-frontend
+  repository: gcr.io/airqo-250220/pr-previews/netmanager-pr-previews
```
Comment (Baalmart) [inline k8s/netmanager/values-dev.yaml:8]: A quick one @123MwanjeMike , aren't we also able to use ENVs here?

File: k8s/platform/values-dev.yaml [hunk 8]
```diff
@@ -5,7 +5,7 @@
 replicaCount: 1
 
 image:
-  repository: gcr.io/airqo-250220/airqo-dev-next-platform
+  repository: gcr.io/airqo-250220/pr-previews/next-platform-pr-previews
```
Comment (Baalmart) [inline k8s/platform/values-dev.yaml:8]: A quick one @123MwanjeMike , aren't we also able to use ENVs here?

File: k8s/netmanager/values-dev.yaml (hunk 7 above)
Comment (123MwanjeMike) [reply k8s/netmanager/values-dev.yaml:8]: Hi @Baalmart 
We unfortunately can't for the kubernetes and helm files

**Pull request merged**